app/styles/modules/_spinner.scss

```diff
@@ -40,7 +40,7 @@
 
   .button-row &,
   .input-row & {
-    @include hidpi-background-image('spinnergrey', 33px 33px);
+    @include hidpi-background-image('spinnerwhite', 33px 33px);
```

Comment:
This does not work on my machine because hidpi-background-image expects both spinnerwhite.png and spinnerwhite@2x.png to be available, only the former is checked in. Can [you use the provided .svg file](https://github.com/mozilla/fxa-content-server/issues/6284#issuecomment-418432439) directly, it will work well at all resolutions. You should be able to replace the `@include` with a simple `background-image`, see [how it's done with device icons](https://github.com/shreya99oak/fxa-content-server/blob/5c4a76ccf3da4d62fedf407dbced5acff8a267d0/app/styles/modules/_settings.scss#L516).

Reply:
I have added the .svg file instead, I had to add the background-size as the image appeared distorted without it